templates/scripts/publish_templates.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+templates_location="$(dirname $(readlink -e $0))/.."
```

Comment:
Might be a good idea to keep this an explicit array of directories.  There are more than just templates within the `templates` directory (`scripts`, `docs`)